internal/probes/mcptransport/ssesession.go

```
@@ -0,0 +1,679 @@
+	if err != nil {
+		a.SetError(err)
+		return nil, a
+	}
+	req.Header.Set("Accept", "text/event-stream")
+
+	resp, err := client.Do(req)
+	if err != nil {
+		a.SetError(err)
+		return nil, a
+	}
+	defer resp.Body.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `resp.Body.Close` is not checked

(errcheck)